packages/common/env/package.json

```diff
@@ -3,8 +3,8 @@
   "private": true,
   "type": "module",
   "devDependencies": {
-    "@blocksuite/global": "0.17.0-canary-202408191538-1511d04",
-    "@blocksuite/store": "0.17.0-canary-202408191538-1511d04",
+    "@blocksuite/global": "0.0.0-canary-20240902070217",
+    "@blocksuite/store": "0.0.0-canary-20240902070217",
```

Comment:
Need an official version

Reply:
I prefer to not add version in snapshot version. And we should stop using snapshot versions in affine.